handbook/people-ops/spending-company-money.md

```diff
@@ -1,3 +1,67 @@
 # Spending company money
+Please see below for a list of reimbursable expenses (all in USD).
+
+## One-time expenses
+**Laptop (engineers):** $3500
```

Comment:
Do note that prices of laptops vary substantially across locations and markets (notable case: Apple). Having a fixed USD price limit for everyone for these things isn't ideal and would result in some degree of unwanted unfairness I believe.

Reply:
For engineers, I would rather define this as minimum specs (e.g. CPU, memory, disk space) which would circumvent @tsenart's fairness issue. Engineers would be free to upgrade beyond minimum specs at their own expense. If this proposal sounds good I can propose what those minimum specs would be.

I have no opinion about how this should work for non-engineers.

Reply:
It's definitely harder to define for engineers (since certain specs here are more important), so I'm okay with making this spec-based for engineers.

Reply:
```suggestion

**Laptop (engineers):** (Choose one)
  - 13" MacBook Pro, 2.8 GHz 4-core, 16 GB, 512 GB SSD
  - 16" MacBook Pro, 2.6 GHz 6‑core, 16 GB, 512 GB SSD
```

Reply:
Are we requiring team mates to use Macs?

Reply:
No. I updated my suggestion.

Reply:
@nicksnyder your suggestion appears to be a significant downgrade though (-$1100), not sure if that was intentional. The configuration linked is $2399, vs $3500 of what we currently spend ([if I understand @mercadon correctly here](https://github.com/sourcegraph/about/pull/434#discussion_r355606217)).
As a data point, as I am typing this comment my SSD is full with 605GB (it's probably `node_modules` 😆 ) and I have 29GB of RAM in use, both more than the spec now in the PR but under $3500 total if configured.

Reply:
Money isn't what is important here (which is why we removed all references to money). What is important is that engineers have a sufficiently powerful machine to develop our application. I based these initial specs on my laptop (I had to start somewhere), which has served me well but I haven't been doing coding for the last ~2 years. I started with a less powerful laptop when I was hired and asked to upgrade because I noticed it was impacting my productivity. If engineers need 32 GB and a 1TB disk to effectively develop our product, then we can definitely change these specs. I'll send out a survey to collect more data from the team.

FWIW I have a 500GB SSD and am using about half of it. In the past 3 years of having this laptop, I think I have cleaned up unused items on my disk once.

Reply:
The current 16" mbp has 1TB per default